lis/plugins/LIS_DAobs_pluginMod.F90

```diff
@@ -118,9 +118,11 @@ module LIS_DAobs_pluginMod
 ! !INTERFACE:
 subroutine LIS_DAobs_plugin
 !EOP
+! Hans Lievens added DA_PF
 #if ( ( defined DA_DIRECT_INSERTION ) || \
       ( defined DA_ENKS )             || \
-      ( defined DA_ENKF ) )
+      ( defined DA_ENKF )             || \
+      ( defined DA_PF ) )     
```

Comment:
(same here) 
Is the DA_PF working? 
Is it a requirement for this pull request about S1 Snow Depth Assimilation? 
If not, it would be better to keep this out here ... just wondering.